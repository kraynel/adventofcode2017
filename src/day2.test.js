const { getChecksum } = require("./day2");

describe("getChecksum", () => {
  it("should compute example checksum", () => {
    expect(
      getChecksum(`5 1 9 5
7 5 3
2 4 6 8`)
    ).toBe(18);
  });
});
