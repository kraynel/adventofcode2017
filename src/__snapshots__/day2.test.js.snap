// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getChecksum should print the answer 1`] = `44670`;
